.github/workflows/Specs.yml

```diff
@@ -30,7 +30,6 @@ jobs:
 
       - name: Run bundle install
         run: |
-          gem install bundler -v "~> 1.17"
```

Comment:
With this change, we'll need to update the CI scripts I believe. At the moment they force an install of 1.17. @jasl @igor-makarov

Reply:
Yeah, better we moving to Bundler 2.x

Reply:
@dcvz Yes, this is another reason why adding M1 CI is better done as you suggested, after #669 and its prerequisites land.